deploy-board/deploy_board/webapp/agent_report.py

```diff
@@ -148,6 +148,7 @@ def gen_report(request, env, progress, sortByStatus="false"):
     deployStats[env['deployId']] = currentDeployStat
 
     for agent in progress["agents"]:
+        agent["accountId"] = hosts_helper.get_accout_id_by_name(request, agent["hostName"])
```

Comment:
do we have host_id in agent? some hosts might not have hostName

Reply:
yes, we have host_id in agents table